### ISSUE TITLE
Expo Router template not working

### ISSUE THREAD

**Issue by sinnrrr** [tamagui/tamagui#2114]
### Current Behavior

Currently, when trying to bootstrap app with Expo Router official template, receiving this error:
```
❯ yarn start
Starting project at /Users/dmytro.soltusyuk/Work/personal/myapp
Error: Cannot find module '@rnx-kit/metro-config'
Require stack:
...
```

Also, `biome.json` is not properly symlinked (broken link), so it is unavailable.

### Expected Behavior

To be able to start the app right away with `yarn start` and `biome.json` to be defined.

### Tamagui Version

```markdown
1.86.6
```

### Platform (Web, iOS, Android)

```markdown
Any
```

### Reproduction

```markdown
npm create tamagui@latest --template expo-router
cd myapp
yarn start
```

**Comment by TimoGoetze** [comment]
aknowledged. same here

**Comment by dsoltysiuk** [comment]
There's lots more errors going on after installing dependencies, just leaving it here for everyone to be informed.

**Comment by natew** [comment]
Fixed in the new version. The template was pulled out of the monorepo where it was working but never got it fully up outside the monorepo, just upgrading Expo to 50 stable along with the other pinned expo deps seemed to fix it, but I did get one error on first load. It seems Metro isn't recognizing the css file emitted, but the logic looks fine - i emit the css file and then resolve back to metro to handle it. Maybe some easy workaround but I'll check again, the logic is very similar to Nativewinds implementation.
